app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp.properties

```diff
@@ -191,7 +191,9 @@ menu.title.no.window = No Window
 # Help menu items
 menu.title.scene.builder.help = Scene Builder Help
 menu.title.show.welcome = Show Welcome Page
-menu.title.check.updates = Check for Update...
+menu.title.check.updates.available = Update available
+menu.title.check.updates.ok = All up-to-date!
```

Comment:
This text out of context can mean many things. We need a better wording like "Updates: Scene Builder is up-to-date", which is longer, but I see that the menu is already wider than expected:

<img width="426" alt="image" src="https://user-images.githubusercontent.com/2043230/155956961-236e9cac-0886-4bb4-8f0d-acefc6f767ae.png">

Reply:
How about "No updates available"?

Also, menu doesn't have that extra space on Linux :shrug: 
![image](https://user-images.githubusercontent.com/437990/167284282-860e0e2d-c986-4f72-a5e3-ad21d795cbd6.png)